packages/opentelemetry-core/src/correlation-context/propagation/HttpCorrelationContext.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Context,
+  CorrelationContext,
+  GetterFunction,
+  HttpTextPropagator,
+  SetterFunction,
+} from '@opentelemetry/api';
+
+import {
+  getCorrelationContext,
+  setCorrelationContext,
+} from '../correlation-context';
+
+export const CORRELATION_CONTEXT_HEADER = 'otcorrelationcontext';
```

Comment:
This header should be `otcorrelations` per the spec PR which was merged today. After this change this PR is mergeable. Sorry this took so long.

Reply:
Changed

Thanks for the review.